Backend/Backend/Services/BookmarkService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Backend.Controllers.Requests;
+using Backend.Infrastructure;
+using Backend.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backend.Services;
+
+public class BookmarkService
+{
+    private readonly AppDbContext _db;
+
+    public BookmarkService(AppDbContext db)
+    {
+        _db = db;
+    }
+    
+    public async Task<List<Bookmark>> GetBookmarks(User user, Guid episodeId)
+    {
+        return await _db.Bookmarks.Where(b => b.UserId == user.Id && b.EpisodeId == episodeId).ToListAsync();
+    }
+
+    public async Task<Bookmark> Add(User user, Guid episodId, BookmarkAddRequest request)
+    {
+        Episode? episode = await _db.Episodes.FirstOrDefaultAsync(e => e.Id == episodId);
```

Comment:
You might want to look into using a [null-coalescing operator (??)](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator).